### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by harjot85**
I have made some structural and grammatical changes to enhance clarity and readability for the readers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,11 +113,11 @@ $ make qemu kvm=no
 
 #### QEMU with KVM
 
-To use QEMU with KVM (kernel-based virtual Machine), which is faster than without KVM, you need a CPU with Intel® Virtualization Technology (Intel® VT) or AMD Virtualization™ (AMD-V™) support. Most systems have this disabled by default, so you may need to reboot, go into the BIOS, and enable it.
+QEMU works faster when used without KVM(kernel-based virtual Machine). To use QEMU with KVM, you need a CPU with Intel® Virtualization Technology (Intel® VT) or AMD Virtualization™ (AMD-V™) support. You may need to reboot, go to the BIOS, and enable this feature, because by default, most systems have this feature disabled.
```
Comment (komawoyo) [inline README.md]: I believe this section needs an update. It's QEMU is faster w/ KVM

**Comment by NeveHanter** [comment]
@harjot85 He actually meant that QEMU is faster with KVM enabled, by incorporating your changes you've negated that information.

**Comment by harjot85** [comment]
Thank you, @NeveHanter, for the clarification. It helped.

File: README.md [hunk 2]
```diff
@@ -75,24 +75,24 @@ The ecosystem and software Redox OS provides is listed below.
...
 
 ## <a name="contributing"> Contributing to Redox </a>
 
-If you're interested in this project, and you'd like to help us out, [here](CONTRIBUTING.md) is a list of ways you can do just that.
+If you're interested in this project, and would like to help, [here](CONTRIBUTING.md) are a few things you can do.
 
 ## <a name="cloning-building-running"> Cloning, Building, and Running </a>
 
-Redox is big, even compressed. Downloading the full history may take a lot of bandwidth, and can even be costly on some data plans. Clone at your own risk!
+Redox is a large operating system, even when it is compressed. Downloading the full history will consume bandwidth, and may cost extra on some data plans. Please check the allowed usage before you clone.
```
Comment (bjorn3) [inline README.md]: Downloading will not just consume bandwidth, it will consume a lot.

File: README.md [hunk 3]
```diff
@@ -113,11 +113,11 @@ $ make qemu kvm=no
 
 #### QEMU with KVM
 
-To use QEMU with KVM (kernel-based virtual Machine), which is faster than without KVM, you need a CPU with Intel® Virtualization Technology (Intel® VT) or AMD Virtualization™ (AMD-V™) support. Most systems have this disabled by default, so you may need to reboot, go into the BIOS, and enable it.
+QEMU is faster when used with KVM(kernel-based virtual Machine) enabled. To use QEMU with KVM, you need a CPU with Intel® Virtualization Technology (Intel® VT), or AMD Virtualization™ (AMD-V™) support. This feature can be enabled in BIOS settings (most systems have this feature disabled). 
 
 ### <a name="manual-setup"> Manual Setup </a>
 
-To manually clone, build and run Redox using a Unix-based host, run the following commands (with exceptions, be sure to read the comments):
+To manually clone, build and run Redox using a Unix-based host, run the following commands. If you recieve exceptions, please go through the comments):
```
Comment (bjorn3) [inline README.md]: I think it is meant that you may (exception) need to adjust the commands.

File: README.md (hunk 2 above)
Comment (le-jzr) [inline README.md]: "Redox is big" -> true,

"Redox is a big operating system" -> false,

Any operating system is gonna be big, but for an operating system, Redox is tiny. The original text better reflects reality.

File: README.md (hunk 3 above)
Comment (peritpatrio) [reply README.md]: There’s a typo: `recieve` should be written as `receive`. 🤓

File: README.md [hunk 4]
```diff
@@ -2,13 +2,13 @@
 <img alt="Redox" width="346" src="https://github.com/redox-os/assets/raw/master/logo.png">
 </p>
 
-**Redox** is an operating system written in Rust, a language with focus on safety and high performance. Redox, following the microkernel design, aims to be secure, usable, and free. Redox is inspired by previous kernels and operating systems, such as SeL4, Minix, Plan 9, and BSD.
+**Redox** is a microkernel operating system written in Rust, a language with focus on safety and high performance. Redox is secure, user-friendly, and free. It is inspired by previous kernels and operating systems, such as SeL4, Minix, Plan 9, and BSD.
```
Comment (vbuell) [inline README.md]: Not sure that I like new version more than the original one

File: README.md [hunk 5]
```diff
@@ -113,11 +113,11 @@ $ make qemu kvm=no
 
 #### QEMU with KVM
 
-To use QEMU with KVM (kernel-based virtual Machine), which is faster than without KVM, you need a CPU with Intel® Virtualization Technology (Intel® VT) or AMD Virtualization™ (AMD-V™) support. Most systems have this disabled by default, so you may need to reboot, go into the BIOS, and enable it.
+QEMU is faster when used with KVM(kernel-based virtual Machine) enabled. To use QEMU with KVM, you need a CPU with Intel® Virtualization Technology (Intel® VT), or AMD Virtualization™ (AMD-V™) support. This feature can be enabled in BIOS settings (most systems have this feature disabled).
```
Comment (vbuell) [inline README.md]: Was better before

**Pull request closed without merging**